tier1/bitbuf.cpp

```diff
@@ -42,14 +42,14 @@ inline unsigned int CountTrailingZeros( unsigned int elem )
 
 inline unsigned int CountLeadingZeros(unsigned int x)
 {
-	uint32 firstBit;
+	unsigned long firstBit;
```

Comment:
What the fuck is that?

Reply:
That's wrong

Reply:
@WitherOrNot, you cant change type to unsigned long, because you broke size of binary read on other platforms. This must be a strictly 32 bit unsigned integer